backend/executor.go

```diff
@@ -282,11 +282,29 @@ func (g *grpcExecutor) PurgeInstances(ctx context.Context, req *protos.PurgeInst
 	if req.GetPurgeInstanceFilter() != nil {
 		return nil, errors.New("multi-instance purge is not unimplemented")
 	}
+	subOrchestrationInstances := make([]api.InstanceID, 0, 10)
```

Comment:
Here we can't set any default values since there are no options passed, similar to `TaskhubClient` and so the default behavior here would be non-recursive Purge(same for terminate as well). Is that okay? Any suggestions to fix?

Reply:
I think we have two options: 1) update all the SDKs to specify recurse = true by default or 2) make the recurse property optional and set the default value to `true` only if no recurse value has been provided by the client (which will be the case for every SDK until they are updated with recurse behavior)

Reply:
@cgillum thoughts on this: https://github.com/microsoft/durabletask-protobuf/pull/20#discussion_r1425672431

The change would be for terminate as well, but with this, we probably won't need any change/work in SDKs?

Reply:
I hesitate a little because this would be a breaking change for existing SDKs. However, I'm not actually aware of whether any stable SDK has taken a dependency on this particular field, so it might be fine to change it.